src/cmd/tools/dtest/docker/harness/resources/coordinator.go

```diff
@@ -57,6 +61,12 @@ var (
 // ResponseVerifier is a function that checks if the query response is valid.
 type ResponseVerifier func(int, map[string][]string, string, error) error
 
+// Sample is a struct containing a metric value at a given timestamp.
+type Sample struct {
+	Timestamp int64
+	Value     float64
+}
```

Comment:
I think you could reuse `ts.Datapoint` (or some other readily available struct).

Reply:
https://github.com/m3db/m3/pull/3065/commits/de8c17f6e067c6426c7e18d7b620c8b8967a62d4#diff-36f3f4899036050c43cca0b378f19429cc8ceb118ebffd0c42f672ee13d507e8R74